release/README.md

```diff
@@ -93,7 +93,7 @@ The releases will have the following version string: `<kubespray-version>-<ck8s-
 
 1. Update public release notes.
 
-    When a released is published the public [user-facing release notes](https://github.com/elastisys/compliantkubernetes/blob/main/docs/release-notes.md) needs to be updated. The new release needs to be added and the list can be trimmed down to only include the supported versions.
+    When a released is published the public [user-facing release notes](https://github.com/elastisys/compliantkubernetes/blob/main/docs/compliantkubernetes/release-notes/kubespray.md) needs to be updated. The new release needs to be added and the list can be trimmed down to only include the supported versions.
```

Comment:
This document does not exist either. But we do have the release notes here https://github.com/elastisys/compliantkubernetes/blob/main/docs/compliantkubernetes/release-notes.md for kubespray. But is it meant to be split now?

Reply:
It does now, I just had it in a PR in the public docs repo which is now merged.